modules/management/src/main/java/org/mule/module/management/agent/DefaultJmxSupportAgent.java

```diff
@@ -116,13 +117,13 @@ public void initialise() throws InitialisationException
             // there must be only one jmx agent, so lookup by type instead
             if (registry.lookupObject(AbstractJmxAgent.class) == null)
             {
-                registry.registerAgent(agent);
```

Comment:
I don't understand. If spring applies the lifecycle correctly, then why is needed to invoke agent.initialise() (line 111)?